build/workflow/Makefile

```diff
@@ -81,6 +83,29 @@ ${TESTS}: setup-testing
 >@${MESSAGE} end "Passed." "Failed."
 >@rm -f status_code
 
+${STANDALONE_MODULE_TEST}: ${MODULE_TESTS}
+>@test_target=$$(echo $@ | sed 's/..\/..\/${TEST_LOCATION}\/${MODULE_NAME}\///g') ; \
+    test_script=$${test_target/test-/} ; \
+    test_name=$${test_script/module_tests\/no_docker_test_/} ; \
+    test_name=$${test_name/unit_tests\/no_docker_test_/} ; \
+    extension=$$(echo $$test_name | grep -o "\.sh\|\.py$$" ) ; \
+    test_name=$${test_name/$$extension/} ; \
+    test_name=$$(echo "$$test_name" | tr '_' ' ' ) ; \
+    ${MESSAGE} start "\u2503 $$test_name"
+>@test_target=$$(echo $@ | sed 's/..\/..\/${TEST_LOCATION}\/${MODULE_NAME}\///g') ; \
+    test_script=$${test_target/test-/} ; \
+    test_name=$${test_script/module_tests\/no_docker_test_/} ; \
+    filename="module_tests/$$(basename $$test_script)" \
+    extension=$${filename##*.} ; \
+    executor=none-cmd ; \
+    if [[ "$$extension" == "py" ]]; then executor=python; fi ; \
+    if [[ "$$extension" == "sh" ]]; then executor=bash; fi ; \
+    cd ../../${TEST_LOCATION}/${MODULE_NAME}/ ; \
+    $$executor $$filename ${DOCKER_ORG_NAME}-development/${DOCKER_REPO_PREFIX}-development:latest ; \
```

Comment:
You don't need to programmatically discover the appropriate executor for this one test -- it's a .sh shell file. You can just use "bash".